src/Microsoft.AspNetCore.Mvc.RazorPages/Internal/PageActionInvokerProvider.cs

```diff
@@ -216,13 +216,10 @@ internal List<Func<IRazorPage>> GetViewStartFactories(CompiledPageActionDescript
                 ViewStartFileName);
             foreach (var item in viewStartItems)
             {
-                if (item.Exists)
```

Comment:
We just need this project system to give us all possible paths. The factoryprovider determines if the view exists